grumblr/db/migrate/20150519205332_add_body_author_to_grumble.rb

```diff
@@ -0,0 +1,6 @@
+class AddBodyAuthorToGrumble < ActiveRecord::Migration
+  def change
+    add_column :grumbles, :body, :string
+    add_column :grumbles, :author, :string
+  end
+end
```

Comment:
interesting, i'm curious as to why you added an additional migration to add columns instead of just including them in your initial migration you created for grumbles.